src/ansible_navigator/actions/_actions.py

```diff
@@ -167,7 +167,7 @@ def run_interactive_factory(package: str) -> Callable:
     return functools.partial(run_interactive, package)
 
 
-def run_stdout(package: str, action: str, *args: Any, **_kwargs: Any) -> Any:
+def run_stdout(package: str, action: str, *args: Any, **_kwargs: Any) -> Tuple[str, int]:
```

Comment:
BTW, I put them in action defs, I'll move and rename AppPublic to the same file.  I did originally have it in action_base, but that introduced a circular import because ActionBase imports kegexes from the action pkg and it would have needed the def from actionbase.